CONTRIBUTING.md

```diff
@@ -1,4 +1,7 @@
+# Contributing
+
 * [Fork](https://help.github.com/articles/fork-a-repo) the project on GitHub.
 * Make your feature addition or bug fix in a feature branch. (Include a description of your changes)
 * Push your feature branch to GitHub.
-* Send a [Pull Request](https://help.github.com/articles/using-pull-requests).
\ No newline at end of file
+* Send a [Pull Request](https://help.github.com/articles/using-pull-requests).
+* Increment the ruby-style-guide version number (i.e. line 3 in **ruby-style-guide.gemspec**)
```

Comment:
2 quick things here:
1. Could you make the gemspec ```[a relative link](ruby-style-guide.gemspec)``` (using this syntax)?
2. What if we add a line to the top of the gemspec? We'd have to update the reference to line 3 ```Increment the ruby-style-guide version number (i.e. the "spec.version" line in [ruby-style-guide.gemspec](ruby-style-guide.gemspec))```